TAKAware/Screens/MainScreens/MapView.swift

```diff
@@ -235,6 +245,15 @@ final class MapPointAnnotation: NSObject, MKAnnotation {
         self.remarks = mapPoint.remarks
         self.videoURL = mapPoint.videoURL
     }
+    
+    init(id: String, title: String, icon: String, coordinate: CLLocationCoordinate2D, remarks: String) {
```

Comment:
Need icon support for KML points. Right now all points would be cloverleafs

Reply:
Created #70 to capture this. For now, defaulting KML points to be Yellow Pushpins